pkg/schedule/schedulers/evict_slow_store.go

```diff
@@ -38,20 +39,42 @@ import (
 )
 
 const (
-	slowStoreEvictThreshold   = 100
-	slowStoreRecoverThreshold = 1
+	slowStoreEvictThreshold = 100
+	// For network slow store, its slow scores include the scores from each tikv.
+	// These scores should meet the following conditions and then will be
+	// considered as a slow store:
+	// 1. One network slow score is greater than networkSlowStoreIssueThreshold.
+	// 2. After removing the maximum value, if the number of stores with a score
```

Comment:
The comment should be changed.

Reply:
done https://github.com/tikv/pd/pull/9451/commits/d520d153ce5e21f6de17cb05f389cad0a822ad32